Models/metrics/alarm.js

```diff
@@ -0,0 +1,11 @@
+const alarm = {

```

Comment:
Look into creating a Javascript class, such that you can instantiate an object by using "new" with parameters. Create these objects in your service before you do insert/update/delete and refer to their attributes